.expeditor/verify.pipeline.yml

```diff
@@ -18,14 +18,14 @@ steps:
   # Lint Tests
 #########################################################################
 
-- label: license-scout
```

Comment:
We can't safely merge without resolving license scout issues.

Reply:
@sreepuramsudheer Please revert this change.